src/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectTypeDescriptionProvider.cs

```diff
@@ -957,7 +969,7 @@ internal static Attribute[] ReflectGetAttributes(Type type)
                 {
                     // Get the type's attributes.
                     //
-                    attrs = type.GetTypeInfo().GetCustomAttributes(typeof(Attribute), false).ToArray();
+                    attrs = type.GetTypeInfo().GetCustomAttributes(typeof(Attribute), false).OfType<Attribute>().ToArray();
```

Comment:
Hmm, ok, thanks for checking.  I must have been looking at a different GetCustomAttributes.